packages/office-ui-fabric-react/src/components/Rating/Rating.Props.ts

```diff
@@ -50,12 +50,28 @@ export interface IRatingProps extends React.AllHTMLAttributes<HTMLElement> {
   ariaLabelIcon?: string;
 
   /**
-   * Optional id of label describing this instance of Rating
+   * Deprecated: Optional id of label describing this instance of Rating. Instead of this use getArialabel
+   * @deprecated
    */
   ariaLabelId?: string;
+
+  /**
+   * Optional flag to mark rating control as readOnly
+   */
+  readOnly?: boolean;
+
+  /**
+   * Optional flag to enable half star support.
+   */
+  enableHalfStar?: boolean;
```

Comment:
Hmmm. Is the idea to default to full start support? Why half? Why not partial? (e.g. 3.2) Or some minimum step amount that defaults to 1?

Reply:
I would love to get this merged! I'd just really like to make sure we don't paint ourselves into a corner here and support rendering whatever step the user may provide. I'm unsure we even need this prop. (e.g. why not just let the user choose to render Math.rounded values?

Reply:
If we remove this, we will end up showing half star in all the existing scenarios where user is passing rating in decimal values without their intention.  Is it fine to break such a behavior with out their intention?

Reply:
Maybe it would be better to have a `step` property then.. default to `1.0` and for you scenario you could pass in `0.5`

Reply:
as discussed offline, removed the prop